src/test/resources/application-test.yml

```diff
@@ -207,4 +209,17 @@ notify:
     cas2assessors: "assessors@example.com"
 
 feature-flags:
-  cas1-use-new-withdrawal-logic: true
\ No newline at end of file
+  cas1-use-new-withdrawal-logic: true
+
+logging:
+  level:
+    org:
+      hibernate:
+        SQL: DEBUG
+        type:
+          descriptor:
+            sql:
+              BasicBinder: TRACE
+    org.springframework.jdbc.core:
+      JdbcTemplate: DEBUG
+      StatementCreatorUtils: TRACE
```

Comment:
did you mean to leave this enabled? Suspect logs could get quite big?

Reply:
Also whoops!